build_resolver.py

```diff
@@ -0,0 +1,230 @@
+"""
+Simple build helper for AYON USD Resolver.
+Automatically detects USD/DCC environment and runs CMake build.
+Adds optional zipping of the install directory.
+"""
+
+import argparse
+import os
+import sys
+import subprocess
+import platform
+import shutil
+import tempfile
+
+
+def run(cmd, cwd=None, env=None):
+    """Run a shell command with logging."""
+    print(f"\n>>> {cmd}")
+    result = subprocess.run(cmd, shell=True, cwd=cwd, env=env)
+    if result.returncode != 0:
+        sys.exit(result.returncode)
+
+
+def detect_houdini_env(root):
+    """Detect paths for Houdini installations."""
+    usd_root = root
+    houdini_cmake_path = f"{usd_root}/toolkit/cmake"
+    python_exec = os.path.join(root, "python", "bin", "python")
+    return [
+        "-DBUILD_TARGET=houdini",
+        f"-DUSD_ROOT={usd_root}",
+        f"-DCMAKE_PREFIX_PATH={houdini_cmake_path}",
+        f"-DPYTHON_EXECUTABLE={python_exec}",
```

Comment:
- `usd_root` variable seems redundant. 
- the return values fail on my side on windows as my paths contain spaces. solution for it was to add double quotes.
- there's no consistency in computing the paths, we use fstring with one and os path join with the other.

```suggestion
    houdini_cmake_path = f"{root}/toolkit/cmake"
    python_exec = os.path.join(root, "python", "bin", "python")
    return [
        "-DBUILD_TARGET=houdini",
        f"-DUSD_ROOT=\"{root}\"",
        f"-DCMAKE_PREFIX_PATH=\"{houdini_cmake_path}\"",
        f"-DPYTHON_EXECUTABLE=\"{python_exec}\"",
```

Reply:
It is still more like help script for running the cmake commands and will be replaced. So I let it like this in this PR...

Reply:
could we create in issue for it?

Reply:
Done